aep/0144.yaml

```diff
@@ -0,0 +1,33 @@
+aliases:
+  UpdateArrayOperation:
```

Comment:
This alias should probably be named `UpdateArrayPathItem` because the result is a PathItem object.